kvdb/etcd/readwrite_cursor.go

```diff
@@ -121,23 +121,11 @@ func (c *readWriteCursor) Seek(seek []byte) (key, value []byte) {
 // invalidating the cursor.  Returns ErrIncompatibleValue if attempted
 // when the cursor points to a nested bucket.
 func (c *readWriteCursor) Delete() error {
-	// Get the next key after the current one. We could do this
```

Comment:
Similar thing here re bolt quirks: IIRC the `Delete` operation always ends up going to the _next_ key as well, which resulted in a bug in the past where we deleted like every other key in the range.

Reply:
merged in: https://github.com/lightningnetwork/lnd/pull/5516